_config.yml

```diff
@@ -15,7 +15,7 @@ description: >
   games on modern PCs.
 baseurl:
 url: "https://xenia.jp"
-twitter_username: xeniaproject
+patreon_username: xenia_project
```

Comment:
It's fine if you want to replace the Twitter link with a Patreon link, but I think the existing code to add a link to Twitter shouldn't be removed. Just comment out the variable that enables it instead.